backend/src/packages/users/user.service.ts

```diff
@@ -20,7 +20,9 @@ class UserService implements IService {
     return Promise.resolve(null);
   }
 
-  public async findByEmail(email: string): Promise<UserAuthResponseDto | null> {
+  public async findByEmail(
+    email: string,
+  ): Promise<Omit<UserAuthResponseDto, 'accessToken'> | null> {
```

Comment:
I believe that this Omit should be moved to a separate type to make it more explicit